packages/server/src/api/controllers/query/import/sources/base/index.ts

```diff
@@ -0,0 +1,105 @@
+export interface ImportInfo {
+  url: string
+  name: string
+}
+
+export interface QueryParameter {
+  name: string
+  default: string
+}
+
+export interface Query {
```

Comment:
This is great, only thing I'd say is could we move the interfaces to the common `src/definitions` directory? Theres a place for datasource definitions in there already. keeping them all centralised like that means that if anyone goes looking for a definition they'll find it there and not think that it needs written etc.

Reply:
Nice suggestion, I wasn't sure on the best place for this. This is ideal - added in https://github.com/Budibase/budibase/pull/3644/commits/9d402d03560f9bbbc4d3106d82aa14904bfba79d